cmd/kube-proxy/app/server_windows.go

```diff
@@ -46,7 +46,11 @@ import (
 )
 
 // NewProxyServer returns a new ProxyServer.
-func NewProxyServer(config *proxyconfigapi.KubeProxyConfiguration, cleanupAndExit bool, scheme *runtime.Scheme, master string) (*ProxyServer, error) {
+func NewProxyServer(o *Options) (*ProxyServer, error) {
```

Comment:
Just double checking, but we're OK to have the "cleanup ipvs" flag (and maybe others) ignored on windows, rather than returning a "not supported" type error?

Reply:
I think that's the easiest path forward.  The flags are kind of a mess, and being cross-platform makes it 10x worse.  We need a more holistic answer